pom.xml

```diff
@@ -76,7 +76,7 @@
         <version.log4j>1.2.16</version.log4j>
         <version.log4j-extras>1.0</version.log4j-extras>
         <version.lucene>7.5.0</version.lucene>
-        <version.microservice.accumulo-utils>1.2</version.microservice.accumulo-utils>
+        <version.microservice.accumulo-utils>1.3-SNAPSHOT</version.microservice.accumulo-utils>
```

Comment:
this can only be for testing purposes.  We need to release a new version of accumulo-utils and then change this to 1.3

Reply:
https://github.com/NationalSecurityAgency/datawave/pull/333#discussion_r256077688